src/Kuzzle.js

```diff
@@ -29,6 +30,7 @@ function Kuzzle (host, options, cb) {
   if (!(this instanceof Kuzzle)) {
     return new Kuzzle(host, options, cb);
   }
+  KuzzleEventEmitter.call(this);
```

Comment:
It's necessary because I nead to call the parent's contructor within Kuzzle's constructor
the line below only applies the parent's prototype to current one, but does not call the eventEmitter's constructor when we instantiate a `Kuzzle` object.

That said, it's the line below that was wrong (or at least not the better syntax). I've just fixed it and replaced it by:
```javascript
Kuzzle.prototype = Object.create(KuzzleEventEmitter.prototype);
```